docs/generated/workloads.md

```diff
@@ -1,925 +1,6 @@
-# Workload Documenation
+# Workload Documentation 
```

Comment:
We need some introductory text here:
* What workloads does this cover?
* What information is available for each workload?
* Where does this documentation come from?
* What to do if documentation is missing/bad?
* ...